README.md

```diff
@@ -1,5 +1,12 @@
 # ASP.NET Core - App Building Workshop
 
+![](https://msft-shayne.visualstudio.com/_apis/public/build/definitions/6bb14994-33a9-43d6-b737-dd68d09428c1/7/badge)
```

Comment:
Badge points to https://msft-shayne.visualstudio.com - should this be the permanent location?

Reply:
Probably not. If we can get a .NET Foundation instance we can move the VSTS Build system there as well as the AppService Instances and ACR etc.

Reply:
Okay, cool. Not a reason to block here, I'm merge and we can add another issue to track.